### PR TITLE
Add blog post about free text responses

### PR DESCRIPTION
**Pull request by edwardhorsford**

### CODE REVIEW

File: app/posts/register-trainee-teachers/2021-04-20-investigating-free-text-responses-from-apply.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+        text: "Master’s Degree"
+      },
+      {
+        text: "6"
+      }
+    ]
+  ]
+}) }}
+
+We can see that most of the remaining free text answers are where the user did not provide enough detail - or perhaps entered a qualification that was not a degree. Where the applicant’s qualification is not a degree it’s unlikely the record would end up in Register - as they would likely not be eligible for the course (for courses that require a degree).For the remaining ones we may need providers to amend the record to give the detail required.
```
Comment (ralph-hawkins) [inline app/posts/register-trainee-teachers/2021-04-20-investigating-free-text-responses-from-apply.md]: missing space in "degree).For"

File: app/posts/register-trainee-teachers/2021-04-20-investigating-free-text-responses-from-apply.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        text: "5"
+      }
+    ]
+    
+  ]
+}) }}
+
+The free text inputs for subjects are largely a mix of subjects which are not listed in the fixed list (English, physical education), or degrees where the candidate has done a mix of subjects (Mathematics and physics).
+
+We’ll look at whether we’re able to add the common subjects that are not in the current degree subject list. We can do the same for the common dual subjects, but there are lots of dual subject combinations. -We need toto do analysis on these to work out what would be the best thing for Register to store.
```
Comment (ralph-hawkins) [inline app/posts/register-trainee-teachers/2021-04-20-investigating-free-text-responses-from-apply.md]: "-We need toto do analysis on these to work out what would be the best thing for Register to store." — typos

**Review by ralph-hawkins — APPROVED** [review]
Minor typos

**Comment by edwardhorsford** [comment]
@ralph-hawkins resolved your comments. Can you look again? thanks!

**Pull request merged**